assets/src/edit-story/components/keyboard/index.js

```diff
@@ -341,16 +341,24 @@ function crossesDialogBoundary(target, keyTarget) {
   return dialog && keyTarget !== dialog && keyTarget.contains(dialog);
 }
 
+/**
+ * Determines if the current platform is a Mac or not.
+ *
+ * @return {boolean} True if platform is a Mac.
+ */
+export function isPlatformMacOS() {
+  const { platform } = global.navigator;
+  return platform.includes('Mac') || ['iPad', 'iPhone'].includes(platform);
```

Comment:
(I like this clean up!) 

nit: would it be useful here to shift everything lowercase? is there ever a chance that platform might have different casing?

Reply:
Good call, we could do it for extra safety, but after doing some research it does appear that platforms do come in with proper casing, so this code should be safe.